code/modules/paperwork/pen.dm

```diff
@@ -139,6 +139,40 @@
 	reagents.add_reagent("ketamine", 100)
 
 
+
+/obj/item/pen/love
+	container_type = DRAINABLE //cannot be refilled, love can be extracted for use in other items with syringe
+	origin_tech = "engineering=4;syndicate=2"
+
+
+/obj/item/pen/love/attack(mob/living/M, mob/user)
+	if(!istype(M))
+		return
+
+	if(!M.can_inject(user, TRUE))
+		return
+	var/transfered = 0
+	var/contained = list()
+
+	for(var/R in reagents.reagent_list)
+		var/datum/reagent/reagent = R
+		contained += "[round(reagent.volume, 0.01)]u [reagent]"
+
+	if(reagents.total_volume && M.reagents)
+		transfered = reagents.trans_to(M, 25) //injects 25u of love, forcing help intent
+		M.apply_status_effect(STATUS_EFFECT_PACIFIED) //pacifies for 40 seconds
```

Comment:
Since this seems to be the only line that's really different, after subtyping you could replace this proc with something like this, sandwiching the original call with logic on either side to see if the thing could be applied.

```
/obj/item/pen/sleepy/love/attack(mob/living/M, mob/user)
    var/can_transfer = reagents.total_volume && M.reagents
    . = ..()
    if (can_transfer && .)
        M.apply_status_effect(STATUS_EFFECT_PACIFIED) //pacifies for 40 seconds